src/classes/REL_Relationships_TDTM.cls

```diff
@@ -218,8 +210,7 @@ public with sharing class REL_Relationships_TDTM extends TDTM_Runnable {
         for(npe4__Relationship__c r : Relationships) {
         	
             //SWA 2009-05-13 removed a duplicate check for change to npe4__Status__c from this OR statement
-            if ((r.npe4__Status__c != oldRelationships[counter].npe4__Status__c 
-            || r.npe4__Type__c != oldRelationships[counter].npe4__Type__c 
+            if ((r.npe4__Type__c != oldRelationships[counter].npe4__Type__c 
```

Comment:
well, either back out this fix and leave the bug open, or we can try to do the correct fix.  I looked a little at the code, and I don't fully understand how the rels originally had gender specific roles, but on re-evaluation, they get reset to neutral.  is it only one of the contacts has gender specified, and the rels were originally started from that gendered contact, but the update to former is happening on the ungendered contact's rel?